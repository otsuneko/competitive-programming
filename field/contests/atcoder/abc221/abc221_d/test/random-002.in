10
4 4
2 5
3 3
7 6
8 1
10 10
4 3
4 4
5 5
10 9
