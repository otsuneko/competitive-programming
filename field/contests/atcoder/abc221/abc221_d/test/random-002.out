2 8 4 1 2 1 0 0 0 0
